docs/pages/guides/migrating-to-react-native-firebase.mdx

```diff
@@ -0,0 +1,154 @@
+
+<Terminal cmd={["$ npx expo install expo-build-properties"]} />
+
+#### Example app.json with config plugin
+
+```json
+{
+  "expo": {
+    "plugins": [
+      [
+        "expo-build-properties",
+        {
+          "ios": {
+            "useFrameworks": "static"
+          }
+        }
+      ]
+    ]
+  }
+}
+```
+
+If you are using the Bare Workflow, follow [React Native Firebase's installation instructions](https://rnfirebase.io/#bare-workflow).
+
+## Update analytics requests
+
+Firebase Analytics requests in React Native Firebase are almost the same- mostly, they are just imported a little differently. Update your code as shown here:
+
+```diff
+// update imports
+- import * as Analytics from "expo-firebase-analytics";
++ import analytics from "@react-native-firebase/analytics";
+
+// update each request to analytics
+- Analytics.logEvent("test_analytics_event", {
+-  additionaParam: "test",
+- });
++ analytics().logEvent("test_analytics_event", {
++  additionaParam: "test",
++ });
+```
+
+`logEvent` is asynchronous, so be sure to use `await` in front of the function call if you are using it in an `async` function.
+
+## Creating a development build
+
+If you have previously used Expo Go for development and testing, at this point, you will need to migrate to a [development build](https://docs.expo.dev/development/introduction/#from-expo-go-to-development-builds), as Expo Go does not include the native code used in React Native Firebase. A development build supports the same workflows as Expo Go, allowing you to develop from your local machine or test updates via a QR code, except it is compatible with all of the native code you include in your standalone app.
+
+[Follow these instructions for creating a development build](https://docs.expo.dev/development/getting-started/). Once you are finished, you should have an **eas.json** file that includes a build profile that looks like this:
+```
+{
+  "build": {
+    "development": {
+      "developmentClient": true,
+      "distribution": "internal"
+    }
+  }
+}
+```
+If you wish to run a development build on an iOS simulator instead of a device, your profile may also include:
+```
+{
+  "ios": {
+    "simulator": true
+  }
+}
+```
+`expo-dev-client` should be present in your **package.json**, as well. Once you run `eas build` with your development build profile and platfrom of choice, you will have an app binary file that you can install on your device or simulator.
+
+If you are already using a development build, just rebuild your app, as your native code dependencies have changed.
+
+## Test your changes
+
+Test your changes locally on your development build by running
+<Terminal cmd={[
+'npx expo start --dev-client'
+]} />
+Then follow the prompts on the terminal to run it on your simulator, or scan the QR code to run it on your device. Try an action that will register a Firebase Analytics event. Note that it may take a few minutes for events to show up in the Firebase Console.
```

Comment:
```suggestion

Then follow the prompts on the terminal to run it on your simulator, or scan the QR code to run it on your device. Try an action that will register a Firebase Analytics event. Note that it may take a few minutes for events to show up in the Firebase Console.
```